Makefile

```diff
@@ -45,6 +45,8 @@ ifneq ($(SKIP_DOCKER),true)
 		-e SKIP_DOCKER=true \
 		-e DOCKER_USERNAME=$${DOCKER_USERNAME} \
 		-e DOCKER_PASSWORD=$${DOCKER_PASSWORD} \
+		-e VERSION="$(VERSION)" \
+		-e COMMIT="$(GIT_VERSION)" \
```

Comment:
I know I was probably the one to put these two lines here, but they aren't needed.

The container that Kaniko runs in doesn't need a `VERSION` or `COMMIT` env var because, thanks to the changes on lines 159 and 160, the Kaniko container is passed the entire build command with version/commit info already set appropriately.

Again-- just trying to cut through some noise.

Reply:
Removed.